mirascope/openai/calls.py

```diff
@@ -100,16 +107,15 @@ def call(
                 https://platform.openai.com/docs/guides/error-codes/api-errors
         """
         kwargs, tool_types = self._setup_openai_kwargs(kwargs)
-        client = OpenAI(api_key=self.api_key, base_url=self.base_url)
-        if self.call_params.langfuse:  # pragma: no cover
-            from langfuse.openai import OpenAI as LangfuseOpenAI
-
-            client = LangfuseOpenAI(api_key=self.api_key, base_url=self.base_url)
-        if self.call_params.wrapper is not None:
-            client = self.call_params.wrapper(client)
-        if self.call_params.logfire:
-            self.call_params.logfire(client)  # pragma: no cover
-        create = client.chat.completions.create
+        client = get_wrapped_client(
+            OpenAI(api_key=self.api_key, base_url=self.base_url), self
+        )
+        create = get_wrapped_call(
```

Comment:
why don't we have a check for `configuration.llm_ops` here and elsewhere like the other calls?

Reply:
just forgot. good catch